benchmarking/simulation/simulation.py

```diff
@@ -54,6 +53,8 @@ def __init__(
             raise ValueError(
                 "Either num_teams OR a project initial_teams_provider must be specified."
             )
+        if len(self.metrics) < 1:
```

Comment:
nit;
I think the _pythonic_ way to do this is more like
```suggestion
        if not self.metrics:
```
Then we handle the None case implicitly too :D

Reply:
Done.